pydis_site/apps/main/views/home.py

```diff
@@ -0,0 +1,106 @@
+import requests
+from django.conf import settings
+from django.shortcuts import render
+from django.utils import timezone
+from django.views import View
+
+from pydis_site.apps.main.models import RepoData
+
+GITHUB_API = "https://api.github.com/users/python-discord/repos"
+
+
+class Home(View):
+
+    def _get_api_data(self):
+        """Call the GitHub API and get information about our repos."""
+        repo_dict = {repo_name: {} for repo_name in settings.HOMEPAGE_REPOS}
+
+        # Fetch the data from the GitHub API
+        api_data = requests.get(GITHUB_API)
```

Comment:
I don't see the value for `requests` here. We could do this just as simply with `urllib` and parsing its body with `json` and wouldn't need the extra dependency. What do you think?

Reply:
You could argue that for _any_ use of requests. I think it's a reasonable dependency to add. This is so far the only webrequest this application makes, but there will undoubtedly be many more in the future and I really don't see any reason not to have them all use `requests`. It's one of the top ten most popular python modules in the world for a reason.